tests/release.tests.conf.js

```diff
@@ -0,0 +1,7 @@
+const BaseConfig = require('./base.conf.js');
+console.log(__dirname);
+const pathToStandardConfig = __dirname + '/../config/standard';
+let baseConfig = new BaseConfig('e2e/release', { headless:false }, pathToStandardConfig);
+baseConfig.SELENIUM_PROMISE_MANAGER = false;
```

Comment:
Oh.. this is why you need async everywhere I guess. What's the advantage of turning this off?

Reply:
Ok, yes, that's much better. Nice one.